--- sexplib-rumprun.112.24.01.orig/setup.ml	2015-03-31 13:37:35.000000000 +0200
+++ sexplib-rumprun.112.24.01/setup.ml	2015-05-20 11:27:09.948451048 +0200
@@ -4155,51 +4155,7 @@
       ~short_desc:(fun () -> s_ "Compiler support generation of .cmxs.")
       ~cli:CLINone
       "native_dynlink"
-      (fun () ->
-         let res =
-           let ocaml_lt_312 () =
-             OASISVersion.comparator_apply
-               (OASISVersion.version_of_string (ocaml_version ()))
-               (OASISVersion.VLesser
-                  (OASISVersion.version_of_string "3.12.0"))
-           in
-           let flexdll_lt_030 () =
-             OASISVersion.comparator_apply
-               (OASISVersion.version_of_string (flexdll_version ()))
-               (OASISVersion.VLesser
-                  (OASISVersion.version_of_string "0.30"))
-           in
-           let has_native_dynlink =
-             let ocamlfind = ocamlfind () in
-               try
-                 let fn =
-                   OASISExec.run_read_one_line
-                     ~ctxt:!BaseContext.default
-                     ocamlfind
-                     ["query"; "-predicates"; "native"; "dynlink";
-                      "-format"; "%d/%a"]
-                 in
-                   Sys.file_exists fn
-               with _ ->
-                 false
-           in
-             if not has_native_dynlink then
-               false
-             else if ocaml_lt_312 () then
-               false
-             else if (os_type () = "Win32" || os_type () = "Cygwin")
-                     && flexdll_lt_030 () then
-               begin
-                 BaseMessage.warning
-                   (f_ ".cmxs generation disabled because FlexDLL needs to be \
-                        at least 0.30. Please upgrade FlexDLL from %s to 0.30.")
-                   (flexdll_version ());
-                 false
-               end
-             else
-               true
-         in
-           string_of_bool res)
+      (fun () -> "false")
 
 
   let init pkg =
