--- ctypes.0.4.1/Makefile.rules	2015-04-06 23:19:34.000000000 +0200
+++ ctypes-rumprun.0.4.1/Makefile.rules	2015-06-11 14:22:00.344932914 +0200
@@ -11,9 +11,9 @@
 endif
 else
 ifeq ($(DEBUG),false)
-  CFLAGS=-fPIC -Wall -O3 $(OCAML_FFI_INCOPTS)
+  CFLAGS=-Wall -O3 $(OCAML_FFI_INCOPTS)
 else
-  CFLAGS=-fPIC -Wall -g $(OCAML_FFI_INCOPTS)
+  CFLAGS=-Wall -g $(OCAML_FFI_INCOPTS)
 endif
 endif
 
@@ -55,8 +55,7 @@
 LIB_TARGETS = $(BUILDDIR)/$(PROJECT).cma \
               $(STUB_LIB) \
               $(XEN_LIB) \
-              $(BUILDDIR)/$(PROJECT).cmxa \
-              $(BUILDDIR)/$(PROJECT).cmxs
+              $(BUILDDIR)/$(PROJECT).cmxa
 LIB_TARGET_EXTRAS = $(if $(STUB_LIB),$(BUILDDIR)/lib$(PROJECT)_stubs.a) \
                     $(if $(XEN_LIB),$(BUILDDIR)/lib$(PROJECT)_stubs_xen.a) \
                     $(BUILDDIR)/$(PROJECT).a
